django_extensions/management/commands/sqlcreate.py

```diff
@@ -70,11 +70,20 @@ def handle(self, *args, **options):
         elif engine in ('postgresql', 'postgresql_psycopg2', 'postgis'):
             if options['drop']:
                 print("DROP DATABASE IF EXISTS %s;" % (dbname,))
-                print("DROP USER IF EXISTS %s;" % (dbuser,))
+                if dbuser:
+                    print("DROP USER IF EXISTS %s;" % (dbuser,))
+
+            if dbuser and dbpass:
+                print("CREATE USER %s WITH ENCRYPTED PASSWORD '%s' CREATEDB;" % (dbuser, dbpass))
+                print("CREATE DATABASE %s WITH ENCODING 'UTF-8' OWNER \"%s\";" % (dbname, dbuser))
+                print("GRANT ALL PRIVILEGES ON DATABASE %s TO %s;" % (dbname, dbuser))
+            else:
+                print(
+                    "-- Assumes that unix domain socket connection mode is being used because "
+                    "USER or PASSWORD is blank in django DATABASES configuration."
```

Comment:
Couple of small suggestions / nitpicks:

- Better to talk with authority and say "-- **Assuming** that"
- Think that should be "USER or PASSWORD **are** blank" ?
- Django with capital D "blank in **D**jango DATABASES"

Reply:
Thanks for the review. Fixed.